packages/web/src/shared/components/form/ReorderableMultiselectInput.tsx

```diff
@@ -0,0 +1,163 @@
+    const addItem = (data: NewItemFormData) => {
+      console.log(data);
+      // if (value) {
+      //   const newValue = [...(value ?? []), itemValue];
+      //   onChange?.(newValue);
+      // }
+    };
+    const moveItem = (from: number, to: number) => {
+      const newValue = [...(value ?? [])];
+      newValue.splice(to, 0, newValue.splice(from, 1)[0]);
+      onChange?.(newValue);
+    };
+    const removeItem = (index: number) => {
+      const newValue = [...(value ?? [])];
+      newValue.splice(index, 1);
+      onChange?.(newValue);
+    };
+    return (
+      <div
+        className={`${className} group/multiselect relative flex flex-col gap-1`}
+      >
+        <div className="border rounded flex-col shadow-inner flex border-slate-400">
+          {(value ?? [])
+            .map((v) => items.find((i) => i.value === v))
+            .filter((item?: ItemType): item is ItemType => !!item)
+            .map((item, i, value) => {
+              const isFirst = i === 0;
+              const isLast = i === value.length - 1;
+              return (
+                <div className="py-2 px-1 flex items-center" key={item.value}>
+                  <button className="w-8 h-8" type="button">
+                    <Icon icon="grip-vertical" />
+                    {/* TODO: use different text */}
+                    <span className="sr-only">{t('common:close')}</span>
+                  </button>
+                  <span className="grow mx-1">{item.label}</span>
+                  <button
+                    className={`w-8 h-8 pt-[2px] ${
+                      isFirst && 'disabled:opacity-25'
+                    }`}
+                    type="button"
+                    disabled={isFirst}
+                    onClick={() => moveItem(i, i - 1)}
+                  >
+                    <Icon icon="chevron-up" />
+                    {/* TODO: use different text */}
+                    <span className="sr-only">{t('common:close')}</span>
+                  </button>
+                  <button
+                    className={`w-8 h-8 pb-[2px] ${
+                      isLast && 'disabled:opacity-25'
+                    }`}
+                    type="button"
+                    disabled={isLast}
+                    onClick={() => moveItem(i, i + 1)}
+                  >
+                    <Icon icon="chevron-down" />
+                    {/* TODO: use different text */}
+                    <span className="sr-only">{t('common:close')}</span>
+                  </button>
+                  <button
+                    className="w-8 h-8"
+                    type="button"
+                    onClick={() => removeItem(i)}
+                  >
+                    <Icon icon="close" />
+                    <span className="sr-only">{t('common:close')}</span>
+                  </button>
+                </div>
+              );
+            })}
+        </div>
+        <Form
```

Comment:
Ah, maybe it wasn't the best idea to have our base components connect to Form components by default if they are nested inside. That makes them less flexible. Maybe we add a boolean prop to the ComboboxInput that defaults to false named `disconnect` or `isolate` that if true, prevents connection to the formContext

Reply:
Ok, that worked. Now I'm getting this annoying warning

```
Warning: Received `true` for a non-boolean attribute `isolate`.
```

We don't get this warning for `fitToScreen` in `View`. What about my `isolate` prop is causing this warning? It IS set to a boolean type (though I'm guess that doesn't matter because TypeScript is only ran at compile time).

Reply:
@arrocke did you see this?

Reply:
Found the issue. In the ComboboxInput component, `props.isolate` is passed to the base component. When that happens, it is also passed to `Combobox.Input`. The error in the browser is because react is trying to set isolate on the input element as an HTML attribute. So if we destructure the props in the component so we don't pass it down to child components, we should be good.